### PR TITLE
Satfilter endpoint

### PR DESCRIPTION
**Pull request by RishiVaya**
added endpoint to get enxt passes by noradid and time for filtering

### CODE REVIEW

File: src/leo-server-app/src/routes/satellite.ts [hunk 1]
```diff
@@ -172,6 +174,43 @@ router.get("/getNextPasses", async (req: any, res: any) => {
...
+    noradId: string;
+    startTime: Date;
+    endTime: Date;
+  };
+};
+
+router.get("/getNextPassesByTime", async (req: GetNextPassesByNoradAndTimeProp, res: any) => {
+  const today = new Date();
+
+  const endT = new Date(today.getTime() + (7*1000 * 60 * 60 * 24));
```
Comment (quinnha) [inline src/leo-server-app/src/routes/satellite.ts:188]: why define endT and then say endTime = endT? why not just make it one line

**Review by quinnha — APPROVED** [review]
one nit but lgtm

**Review by Umang-Rajkarnikar — APPROVED** [review]
lgtm

**Review by DiamondAhuja — APPROVED** [review]
lgtm

**Pull request merged**